pkgs/by-name/so/sou/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+  versionCheckHook,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "sou";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "knqyf263";
+    repo = "sou";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-uGYCmW60OvEfserujQMXC9r8S8W+EN+w9EXUGjk1vtw=";
+  };
+
+  vendorHash = "sha256-6kgiZx/g1PA7R50z7noG+ql+S9wSgTuVTkY5DIqeJHY=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X=main.version=${finalAttrs.version}"
+  ];
+
+  doInstallCheck = true;
+  nativeInstallCheck = [ versionCheckHook ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Tool for exploring files in container image layers";
+    homepage = "https://github.com/knqyf263/sou";
+    license = lib.licenses.asl20;
+    maintainers = with lib.maintainers; [ nicolas-goudry ];
```

Comment:
personal nit: i really like having single maintainers as
```
    maintainers = [ lib.maintainers.nicolas-goudry ];
```

Reply:
Feels weird to disagree and just "ignore" you. I personally prefer the contrary (ie. always having `with lib.maintainers`) so that if an additional maintainer were to be added, they only have to add their name to the list. Therefore, I’ll keep the code as is.